test/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -33,6 +34,34 @@
     
     private static final LocalDateTime LOCAL_DATE_TIME = LocalDateTime.parse("2010-10-10T10:10:10.111");
     
+    @Test
+    public void parseAccountId() {
+        AccountId accountId = BridgeUtils.parseAccountId("test", "identifier");
+        assertEquals("test", accountId.getStudyId());
+        assertEquals("identifier", accountId.getId());
+        
+        accountId = BridgeUtils.parseAccountId("test", "externalid:identifier");
+        assertEquals("test", accountId.getStudyId());
+        assertEquals("identifier", accountId.getExternalId());
+        
+        accountId = BridgeUtils.parseAccountId("test", "externalId:identifier");
+        assertEquals("test", accountId.getStudyId());
+        assertEquals("identifier", accountId.getExternalId());
```

Comment:
Minor: We should make the identifier have some upper case letters, so we can verify we don't also accidentally lower case the identifier. (Here and for healthCode.)

Reply:
changed identifier to IdentifierA9